src/main/java/Duke.java

```diff
@@ -1,10 +1,184 @@
+import java.util.Scanner;
+
+
 public class Duke {
+    private static Task[] taskList = new Task[100];
+    private static String[] storeUserText = new String[100];
+    private static boolean doExit = false;
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        lineSeparator();
```

Comment:
Method name should be a verb followed by a noun, for example, getLineSeparator() or printLineSeparator()

Reply:
agree